test/external/handle-filter

```diff
@@ -2,20 +2,19 @@
 require 'sensu-handler'
 
 class Filter < Sensu::Handler
-
   def handle
     puts "Event: #{@event.inspect}"
   end
 
-  def api_request(*args)
+  def api_request(*)
```

Comment:
I'm gonna disagree with rubocop, I don't think this makes it any better and I'd argue its less readable.